src/Networking/v2/Models/LoadbalancerStat.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\OperatorResource;
+
+/**
+ * Represents Neutron v2 Loadbalancer Stats
+ *
+ * @property Api $api
+ */
+class LoadbalancerStat extends OperatorResource implements Retrievable
+{
+    /**
+     * @var integer
+     */
+    public $bytesIn;
+
+    /**
+     * @var integer
```

Comment:
Yeah I agree with @haphan. I think `string` is fine for readonly data, because the alternative would involve using a third party dependency for 2 fields. If folks want to do arithmetic on the value, they can use whatever library they want, but I don't think this SDK should enforce that choice on other users.